### PR TITLE
add a simple implementation of `solarkraft list`

### PR DESCRIPTION
**Pull request by konnov**
This PR implements `solarkraft list`. Since our file storage is very simple, the implementation is also simple.

### CODE REVIEW

File: solarkraft/src/fetcher/storage.ts [hunk 1]
```diff
@@ -128,6 +144,41 @@ export function loadContractCallEntry(filename: string): ContractCallEntry {
...
+ * Generate storage entries for a given contract id in a path.
+ * @param contractId contract identifier (address)
+ * @param path the path to the contract storage
+ */
+export function* yieldListEntriesForContract(
+    contractId: string,
+    path: string
+): Generator<ListEntry> {
+    for (const dirent of readdirSync(path, { withFileTypes: true })) {
+        if (dirent.isDirectory() && /^[0-9]+$/.exec(dirent.name)) {
```
Comment (Kukovec) [inline solarkraft/src/fetcher/storage.ts:157]: Please add a comment explaining this regex

File: solarkraft/src/fetcher/storage.ts [hunk 2]
```diff
@@ -128,6 +144,41 @@ export function loadContractCallEntry(filename: string): ContractCallEntry {
...
+            // This directory may contain several transactions for the same height.
+            const height = Number.parseInt(dirent.name)
+            for (const ledgerDirent of readdirSync(join(path, dirent.name), {
+                withFileTypes: true,
+            })) {
+                // match all storage entries
+                const matcher = /^entry-([0-9a-fA-F]+)\.json$/.exec(
```
Comment (Kukovec) [inline solarkraft/src/fetcher/storage.ts:164]: Why are we using mixed-case HEX here?

**Review by Kukovec — APPROVED** [review]
LGTM

File: solarkraft/src/fetcher/storage.ts (hunk 2 above)
Comment (konnov) [reply solarkraft/src/fetcher/storage.ts:164]: I would be ok with lowercase, but neither Windows, nor MacOS agree with me on the capitalization of filenames

Comment (konnov) [reply solarkraft/src/fetcher/storage.ts:165]: added a comment

**Pull request merged**